controllers/glanceapi_controller.go

```diff
@@ -979,3 +1014,58 @@ func (r *GlanceAPIReconciler) ensureDeletedEndpoints(
 	}
 	return ctrl.Result{}, err
 }
+
+func (r *GlanceAPIReconciler) ensureImageCacheJob(
+	ctx context.Context,
+	h *helper.Helper,
+	instance *glancev1.GlanceAPI,
+	serviceLabels map[string]string,
+	serviceAnnotations map[string]string,
+	cjType glance.CronJobType,
+) (ctrl.Result, error) {
+
+	var err error
+	var ctrlResult ctrl.Result
+
+	command := glance.GlanceCacheCleaner
+	command = glance.GlanceCacheCleaner
+	schedule := glance.CacheCleanerDefaultSchedule
+	// TODO: Fix debug option depending on the API
+	debugArg := ""
+
+	if cjType == "pruner" {
```

Comment:
Could use `glance.CachePruner` instead of hardcoded string?

Reply:
yeah, we have `const` for this purpose, updating that part!